packages/fxa-settings/src/pages/SigninReported/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { getFtlBundle, testL10n } from 'fxa-react/lib/test-utils';
+import { FluentBundle } from '@fluent/bundle';
+import SigninReported from '.';
+import { usePageViewEvent } from '../../lib/metrics';
+
+jest.mock('../../lib/metrics', () => ({
+  usePageViewEvent: jest.fn(),
+}));
+
+describe('SigninReported', () => {
+  let bundle: FluentBundle;
+  beforeAll(async () => {
+    bundle = await getFtlBundle('settings');
+  });
+  it('renders Ready component as expected', () => {
+    render(<SigninReported />);
+    const ftlMsgMock = screen.getAllByTestId('ftlmsg-mock')[1];
```

Comment:
Just curious, why was `[1]` needed here?

Reply:
We do! I will add a comment with the next PR -- this was part of the hack to make the l10n tests pass (until they get fixed)